README.md

```diff
@@ -68,93 +68,132 @@ services:
 {}
 ```
 
-Let's edit the parameters section and add the following default values for our application:
+Let's edit the parameters section and add the following default values for our
+application:
 
 ```yaml
 port: 5678
 text: hello development
-version: latest
 ```
 
-Then modify the Compose file section in `hello.dockerapp`, adding in the variables.
+Then modify the Compose file section in `hello.dockerapp`, adding in the
+variables.
 
 ```yaml
 version: '3.2'
 services:
   hello:
-    image: hashicorp/http-echo:${version}
+    image: hashicorp/http-echo
     command: ["-text", "${text}"]
     ports:
       - ${port}:5678
 ```
 
-Finally you can test everything is working, by rendering the Compose file with the provided default values.
+You can test everything is working, by inspecting the application.
```

Comment:
the Application Package?

Reply:
or definition ?

Reply:
We're using "definition" or "source" for the on disk representation and "package" for the image format.